### PR TITLE
fix(36068): Incorrect quick fix for undeclared private field i…

### LINKED ISSUE #36068: Incorrect quick fix for undeclared private field in javascript files
**TypeScript Version:** 3.8.0-dev.20200108

**Search Terms:**

**Code**

For a js file

```js
//@ts-check

class Private {
    constructor(name) {
        this.#name = name;
    }
}

```

Trigger the `initialize property name` quick fix

**Expected behavior:**
Initializer added in class:

```js
//@ts-check

class Private {
    #name;
    
    constructor(name) {
        this.#name = name;
    }
}
```

**Actual behavior:**
Initializer added in the ctor:

```js
//@ts-check

class Private {
    constructor(name) {
        this.#name = name;
        this.#name = undefined;
    }
}
```

**Playground Link:** 

**Related Issues:** 
- #35962

### PR DESCRIPTION
**Pull request by a-tarasyuk**
Fixes #36068

### CODE REVIEW

File: tests/cases/fourslash/codeFixInitializePrivatePropertyJS.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+verify.codeFixAvailable([
+    { description: "Initialize a private property '#name'" },
+    { description: "Ignore this error message" },
+    { description: "Disable checking for this file" },
+    { description: "Infer parameter types from usage" },
+]);
+
+verify.codeFix({
+    index: 0,
+    description: "Initialize a private property '#name'",
```
Comment (DanielRosenwasser) [inline tests/cases/fourslash/codeFixInitializePrivatePropertyJS.ts]: Can you reference `ts.DiagnosticMessages.[...].text` here?

File: src/compiler/diagnosticMessages.json [hunk 2]
```diff
@@ -5056,6 +5056,10 @@
         "category": "Message",
         "code": 90034
     },
+    "Initialize a private property '{0}'": {
```
Comment (DanielRosenwasser) [inline src/compiler/diagnosticMessages.json]: It's not really the initialization, it's more the declaration

```
Declare private field '{0}'.
```

File: tests/cases/fourslash/codeFixInitializePrivatePropertyJS.ts (hunk 1 above)
Comment (a-tarasyuk) [reply tests/cases/fourslash/codeFixInitializePrivatePropertyJS.ts]: @DanielRosenwasser done

File: src/compiler/diagnosticMessages.json (hunk 2 above)
Comment (a-tarasyuk) [reply src/compiler/diagnosticMessages.json]: @DanielRosenwasser Changed.

**Comment by DanielRosenwasser** [comment]
Thank you!

**Pull request merged**